package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "scripts": {
     "bootstrap": "npx lerna bootstrap",
-    "prepare": "lerna run prepare --stream --concurrency=1",
+    "prepare": "lerna run prepare --stream --concurrency=1 && husky install",
```

Comment:
Can we just put this `husky install` step in our README so it doesn't run every time?

Reply:
Since we discussed this on our 1/1 I'm ok to let it stand because husky's documented way to configure itself is in the prepare life-cycle.